actioncable/lib/rails/generators/channel/templates/assets/channel.js

```diff
@@ -0,0 +1,21 @@
+(function() {
+  App.<%= class_name.underscore %> = App.cable.subscriptions.create("<%= class_name %>Channel", {
+    connected: function() {
+      // Called when the subscription is ready for use on the server
+    },
+
+    disconnected: function() {
+      // Called when the subscription has been terminated by the server
+    },
+
+    received: function(data) {
+      // Called when there's incoming data on the websocket for this channel
+    },
+
+<% actions.each do |action| -%>
+    <%=action %>: function() {
+      return this.perform('<%= action %>');
+    }
```

Comment:
Updated the code. Here is sample output https://gist.github.com/prathamesh-sonpatki/b71262858e609ecc3e64504a42ee7132